src/applications/mhv/medical-records/actions/allergies.js

```diff
@@ -12,10 +12,26 @@ export const getAllergiesList = () => async dispatch => {
   }
 };
 
-export const getAllergyDetails = id => async dispatch => {
+export const getAllergyDetails = (id, allergyList) => async dispatch => {
   try {
-    const response = await getAllergy(id);
-    dispatch({ type: Actions.Allergies.GET, response });
+    // Check if allergyList has data
+    if (allergyList && allergyList.length > 0) {
```

Comment:
What would a user need to do to refresh this list?

Reply:
list provided by state